### ISSUE TITLE
T-0120 - Rung 5: noisy-OR/LSE pooling MIL over payer capsules

### ISSUE THREAD

**Issue by Ayush-3103-AI** [Ayush-3103-AI/Rakshak-RazorPay#54]
## Parent

#51

## What to build

Rung 5 — multiple-instance learning by pooling, not attention (ADR-V3-001 holds: no torch this cycle). The bag is the merchant-day; the instances are the (payer, day) capsules from #119. An instance-level LightGBM scores each capsule; the bag score is a pooled aggregate:

```
noisy-OR :  s(bag) = 1 - Π_i (1 - p_i)
LSE      :  s(bag) = (1/τ) · log( (1/n) Σ_i exp(τ · s_i) )
```

τ is the one fitted parameter, selected on validation. τ → ∞ recovers max-pooling (classic MI: a bag is positive if any instance is); τ → 0 recovers mean-pooling (what the hand-built feature register has effectively been assuming). The instance model trains by alternating bag-label propagation with the pooled loss — two passes maximum, this is a rung, not a research programme.

**Reporting the fitted τ is itself a result.** It says where this problem sits on the any-instance-to-all-instances axis, tested rather than assumed.

**Expected outcome, declared in advance:** the population has no cross-merchant graph signal (cut in #32 §8; mule-ring typology removed as consequently undetectable) and capsules are within-merchant only, so it is entirely possible MIL adds nothing over the T1 register. A fitted τ near zero is that finding, not a failure of the ticket.

## Acceptance criteria

- [ ] τ → ∞ recovers max-pooling and τ → 0 recovers mean-pooling to numerical tolerance (proves the pooling is provably the family it claims)
- [ ] A bag with one strongly positive instance scores above an otherwise-identical bag with no positive instances
- [ ] Rung 5 registers as a `Scorer` and is reported against the same floors and the same oracle as every other rung (`oracle_savings >= rung5_savings` holds)
- [ ] The fitted τ is reported in the results table, not just logged
- [ ] Trains on train+validation only; does not touch the test split

## Blocked by

- #119 (T-0119)
- #47 (T-0113) — needs Rung 2 as the LightGBM baseline it is compared against

**Comment by Ayush-3103-AI** [comment]
Removed `blocked`: the blocking prerequisite ticket is now done, but this ticket's own work is still unfinished per its own commit's notes. Blockers #53/#47 are done, but 17ea616 itself states: 'Left undone: no fitted tau exists... nobody has run it on real bags.' Stays open.

**Comment by Ayush-3103-AI** [comment]
Verified done. `src/rakshak/models/rung5_mil.py` built, scored 5 seeds on cycle-4 data (`data/v2/eval/rung5_mil_val_seed4{2..6}.json`), reported in LIMITATIONS.md §9.9 — best PR-AUC on the ladder (0.7836), near-worst savings, a real calibration finding. Note: `configs/rung_roster.yaml` still says `status: planned` for this rung — that's tracked separately as a documentation gap under #72, not a reason to reopen this ticket.
